cmd/init.go

```diff
@@ -1,35 +1,78 @@
 package commands
 
 import (
-	"os"
-
+	"github.com/SAP/cloud-mta-build-tool/internal/artifacts"
+	"github.com/SAP/cloud-mta-build-tool/internal/exec"
+	"github.com/SAP/cloud-mta-build-tool/internal/tpl"
 	"github.com/spf13/cobra"
+	"os"
+)
 
-	"github.com/SAP/cloud-mta-build-tool/internal/tpl"
+const (
+	makefile = "Makefile.mta"
 )
 
 // flags of init command
 var initCmdSrc string
 var initCmdTrg string
 var initCmdDesc string
+var initCmdName string
 var initCmdMode string
 
+// flags of build command
+var buildProjectCmdExecFunc = exec.Execute
+var buildProjectCmdSrc string
+var buildProjectCmdTrg string
+var buildProjectCmdDesc string
+var buildProjectCmdName string
+var buildProjectCmdMode string
+var buildProjectCmdMtar string
+var buildProjectCmdPlatform string
+var buildProjectCmdStrict bool
+
 // init flags of init command
 func init() {
+	// set flags of init command
 	initCmd.Flags().StringVarP(&initCmdSrc, "source", "s", "", "Provide MTA source")
 	initCmd.Flags().StringVarP(&initCmdTrg, "target", "t", "", "Provide MTA target")
 	initCmd.Flags().StringVarP(&initCmdDesc, "desc", "d", "", "Descriptor MTA - dev/dep")
+	initCmd.Flags().StringVarP(&initCmdDesc, "name", "n", "", "Name of Makefile")
 	initCmd.Flags().StringVarP(&initCmdMode, "mode", "m", "", "Mode of Makefile generation - default/verbose")
+
+	// set flags of build command
+	buildCmd.Flags().StringVarP(&buildProjectCmdSrc, "source", "s", "", "Provide MTA source")
```

Comment:
please adjust all commands and their flags descriptions with other commands. see branch "init_flags"